src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py

```diff
@@ -114,6 +114,8 @@ def models(cls, api_version=DEFAULT_API_VERSION):
            * 2019-10-01: :mod:`v2019_10_01_.models<azure.mgmt.containerservice.v2019_10_01.models>`
            * 2020-01-01: :mod:`v2020_01_01_.models<azure.mgmt.containerservice.v2020_01_01.models>`
            * 2020-02-01: :mod:`v2020_02_01_.models<azure.mgmt.containerservice.v2020_02_01.models>`
+           * 2020-03-01: :mod:`v2020_03_01_.models<azure.mgmt.containerservice.v2020_03_01.models>`
```

Comment:
We have 2020-03-01?

Reply:
since we don't use 2020-03-01, I think it is not necessary to import this api version.

Reply:
sorry for confusing. 03-01 is already in master. just rebased.